spec/spec_helper.rb

```diff
@@ -27,17 +27,17 @@
 require "rspec/longrun"
 #require "narray/narray"
 
-require "../lib/nmatrix"
-require "../lib/nmatrix/rspec"
+require "./lib/nmatrix"
+require "./lib/nmatrix/rspec"
```

Comment:
No, I squashed the latest 7 commits.

I didn't revert them as it was messing up my commit history.